dss-server/src/test/java/io/github/ztkmkoo/dss/server/rest/DssRestServerTest.java

```diff
@@ -171,4 +235,28 @@ private static File loadFromTestResources(String path) throws UnsupportedEncodin
         final ClassLoader classLoader = DssRestServerTest.class.getClassLoader();
         return new File(URLDecoder.decode(classLoader.getResource(path).getFile(),"UTF-8"));
     }
+
+    private static void startOnDssJsonRestServer(DssRestServer dssRestServer) {
+        startOnNewThread(() -> {
+            dssRestServer
+                    .addDssRestService(new TestJsonService("test_json", "/test/json", DssRestMethodType.POST));
+            try {
+                dssRestServer.start();
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        });
+    }
```

Comment:
I think this method is not general.

It depends on specific conditions.

It does not seem appropriate to write the dssRestSer.addDssRestService method here.